ci-scripts/release-tests.sh

```diff
@@ -36,8 +41,27 @@ export RHDH_HELM_REPO='$RHDH_HELM_REPO'
 
     git commit -am "chore($ticket): $testname on $branch"
     git push -u origin "$branch"
-    echo "$(date -Ins --utc) INFO Create PR on https://github.com/redhat-performance/backstage-performance/pull/new/${branch} and comment '/test mpc' there"
-    git checkout main
+    echo "$(date -Ins --utc) INFO Created and pushed branch ${branch}"
+    git checkout "$SOURCE_BRANCH"
+
+    curl_data='{
+        "title": "chore('"$ticket"'): '"$branch"'",
+        "body": "**'"$testname"'**: '"$VERSION_OLD"' vs. '"$VERSION_NEW"' testing. This is to get perf&scale data for `'"$branch"'`",
+        "head": "'"$branch"'",
+        "base": "main",
```

Comment:
@jhutar base branch shlold not always be `main`
```suggestion
        "base": "'"$SOURCE_BRANCH"'",
```